utils/dyn-config.c

```diff
@@ -537,16 +540,38 @@ glusterBlockDynConfigStart(void *arg)
     return NULL;
   }
 
-  wd = inotify_add_watch(monitor, cfg->configPath, IN_ALL_EVENTS);
+  if (GB_ALLOC_N(cfgDir, PATH_MAX) < 0) {
+    LOG("cmdlog", GB_LOG_ERROR, "%s", "Could not allocate memory for cfgDir\n");
+    return NULL;
+  }
+
+  snprintf(cfgDir, PATH_MAX, cfg->configPath);
```

Comment:
Right now the configPath is not dynamic or modifiable, so its okay to just  use GB_DEF_CONFIGDIR and GB_DEF_CONFIGPATH  instead of collecting it from cfg->configPath. right ?

Reply:
The logdir dyn could work, but /var/log/gluster-block/gluster-block-gfapi.log won't work for now.
So I will remove the logDir dyn patches  and let's add this later.